styles/styles.css

```diff
@@ -0,0 +1,337 @@
+    font-weight: bold;
+    font-size: 75%;
+    line-height: 30px;
+    text-align: center;
+    margin: 10px;
+}  
+
+h1 {
+  
+/*Size */
+}
+
+#subbanner {
+  background-color: #ebebeb;
+  height: 4em;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  align-items: center;
+}
+
+.subtitle {
+ /*Position away from left margin and check font */
+  color: #093b60;
+  font-family: "Archivo Black", Geneva, sans-serif;
+  font-weight: bold;
+  margin-left: 10rem;
+}
+
+#welcome {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: center;
+  margin: 1rem;
+}
+
+#hugpicture {
+  width: 85%;
+}
+
+h2 {
+  font-family: "Palatino Linotype", "Book Antiqua", Palatino, serif;
+  color: #093b60;
+  font-size: 1.75em;
+  font-weight: 900;
+}
+
+.lead-in {
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+  color: #3C3C3C;
+  width: 30rem;
+
+}
+
+.profiles {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: space-around;
+  align-items: center;
+  justify-content: center;
+}
+
+.profile-card {
+  width: 200px;
+  height: 375px;
+  margin: 25px;
+  border: 2px solid #ebebeb;
+}
+
+.profilepicture {
+  width: 200px;
+}
+
+h3 {
+  font-family: "Palatino Linotype", "Book Antiqua", Palatino, serif;
+  color: #093b60;
+  font-size: 1.25em;
+  font-weight: 900;
+}
+
+.cost {
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+  font-size: .8em;
+  color: #3C3C3C;
+}
+
+.bio {
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+  font-size: .8em;
+  color: #3C3C3C;
+}
+
+.adopt-button {
+  background-color: #81B642;
+  height: 30px;
+  width: 180px;
+  color: white;
+  font-family: "Archivo Black", Geneva, sans-serif;
+  font-weight: bold;
+  font-size: 90%;
+  line-height: 30px;
+  text-align: center;
+  margin: 10px;
+}
+
+a.checkout:visited {
+  color: white;
+}
+
+/*Not working: make adopt-button white and still white when visited*/
+
+.superfooter {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: space-around;
+  align-items: center;
+  justify-content: center;
+  height: 10rem;
+}
+
+#whitespace {
+  height: 8rem;
+  /*Make big enough for some padding*/
+
+}
+
+.moredogs {
+  background-color: #ebebeb;
+  height: 50px;
+  width: 225px;
+  color: #093b60;
+  font-family: "Archivo Black", Geneva, sans-serif;
+  font-weight: bold;
+  line-height: 50px;
+  text-align: center;
+  margin: 10px;
+  /*Text blue, background light grey, LR centered, closer to top than bottom*/
+}
+
+a.more:visited {
+  color: #093b60;
+}
+
+#footer {
+  /*Maybe nothing needed*/
+}
+
+.basefield {
+  background-color: #3C3C3C;
+  height: 23rem;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  /*some padding above*/
+}
+
+#bottomlogo {
+  /*size and center LR*; just shrank a bit for now*/
+  width: 22%;
+  
+}
+
+#footer-text {
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+  color: white;
+}
+
+.footer-link {
```

Comment:
Your flexbox container is forcing these into columns. If you put a wapper on these elements, it'll still stack the link container, and then you can layout the links